torchani/utils.py

```diff
@@ -181,9 +184,14 @@ def sae(self, species):
             :class:`torch.Tensor`: 1D vector in shape ``(conformations,)``
             for molecular self energies.
         """
+        intercept = 0.0
+        present_species_ = present_species(species)
+        if self.self_energies.numel() == present_species_.numel() + 1:
```

Comment:
I think the problem is here:
Consider CH3OH, then `present_species_.numel()` will be `3`, but there should not be intercept.

Reply:
Wow! good catch! I will work on it, probably making `fit_intercept` an instance variable.